pkg/build/admission/strategyrestrictions/admission.go

```diff
@@ -157,8 +168,22 @@ func (a *buildByStrategy) checkAccess(strategy buildapi.BuildStrategy, subjectAc
 	if err != nil {
 		return admission.NewForbidden(attr, err)
 	}
+	// If not allowed, try to check against the legacy resource
+	// FIXME: Remove this when the legacy API is deprecated
```

Comment:
That's a long way away.

Why doesn't subjectaccessreview allow multiple resources?